templates/zerver/integrations.html

```diff
@@ -1009,6 +1008,80 @@
 
     </div>
 
+    <div id="gcal" class="integration-instructions">
+
+        <p>Get Google Calendar reminders in Zulip! This is a great way to see your reminders directly in
```

Comment:
If markups (opening and closing ones) and text are not in the same line please use following formatting:
```
<markdup>
    text line1
    text line 2
</markup>